.release-please-manifest.json

```diff
@@ -0,0 +1,4 @@
+{
+  "providers/openfeature-flagd-provider": "0.0.1",
+  "providers/openfeature-sdk-meta_provider": "0.0.1"
```

Comment:
This maps to the folder structure. We should try and align on a naming convention before pushing the Gems.